core/src/main/java/org/apache/gravitino/listener/EventBus.java

```diff
@@ -35,25 +37,45 @@ public class EventBus {
   // which are meant for synchronous event listening, or AsyncQueueListener, designed for
   // asynchronous event processing.
   private final List<EventListenerPlugin> postEventListeners;
+  // Only contain sync EventListenerPlugins.
+  private final List<EventListenerPlugin> preEventListeners;
```

Comment:
Why do we need to have 2 listeners for pre and post event separately?

Reply:
removed